torch_glow/tests/utils.py

```diff
@@ -16,10 +16,12 @@ def jitVsGlow(f, *inputs, expected_fused_ops):
     jitVsGlow_(f, f, *inputs, expected_fused_ops=expected_fused_ops)
 
 
-def jitVsGlow_(f_torch, f_glow, *inputs, expected_fused_ops):
-    assert (
-        expected_fused_ops is not None and len(expected_fused_ops) > 0
-    ), "Must pass non-empty list of ops that are expected to be fused"
+def jitVsGlow_(f_torch, f_glow, *inputs, expected_fused_ops=None):
+
+    accept_all_ops = False
+    if (expected_fused_ops == None):
+        expected_fused_ops = []
+        accept_all_ops = True
```

Comment:
I don't think this is a good idea. The point of expected_fused_ops is to check that the things that should be getting fused are getting fused. Why do we need to have this wildcard?

Reply:
It is because we want to use jitVsGlow not only here in the unit test, but also in, like, testing xray (actually I did have a script testing xray model locally using jiVsGlow). we cannot indicate all ops in a big model like xray easily.
My thinking is, this functionality is like an extra. If we want to check it, then good we will have it to be checked. Else we just dont check it.

Reply:
Ok well I guess my two thoughts are that
1. I'd like to make sure all the operator tests use this so having operator checking be on by default (as opposed to this where it's basically off by default) would be preferred (maybe we can pass an extra bool flag to disable this?)
2. Shouldn't we be checking the ops in the xray model anyways? We want to make sure we're running what we expect to be running.

Reply:
I agree with you for both two opinions. So our decision is:
1. A seperate by_default=False param to control if all ops should be accepted.
2. Once we have an official bigger model unit tests in our code base, we should also have a list of expected ops ( of course future operator unit tests should have this as well)

Reply:
That sounds good to me